website/content/api-docs/variables/locks.mdx

```diff
@@ -0,0 +1,210 @@
+---
+layout: api
+page_title: Variable Locks - HTTP API
+description: The /var endpoints are used to query for and interact with variables and locks.
+---
+
+# Locks HTTP API
+
+The `/var` endpoint is used hold, renew and release a lock over a variable.
+
+## Lock Variable
+
+The endpoint to create a variable can also be used to hold a lock and interact with
+it throught the use of a parameter defining the operation to be performed.
+
+| Method | Path                                 | Produces           |
+|--------|--------------------------------------|--------------------|
+| `PUT`  | `/v1/var/:var_path?<lock-operation>` | `application/json` |
+
+### Parameters
+
+The lock operation parameter can be:
+
+- `lock-acquire`: When used, the call will introduce a lock over the variable if 
+it exists, or create a new one if it doesn't. The lock ID will be returned in the 
+response and it must be provided to perform any other operation over the lock. 
+The variable items can be udpated at any time using the lock ID, but the lock
+parametes are unmmutable, attempting to modify them while a lock is present will
```

Comment:
my brain immediately thinks "immutable" instead of "unmutable" here but maybe there's a subtle difference I'm unaware of.